packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.scss

```diff
@@ -1,36 +1,29 @@
-.dc-modal {
-    $strategy-modal: 'modal--strategy';
-
-    &__container {
-        &_#{$strategy-modal} {
-            max-height: 57.1rem !important;
+.quick-strategy {
+    $quick-strategy: &;
 
-            .modal__scrollbar {
-                max-height: 516px;
-                margin-bottom: 75px;
-            }
-            .modal__content {
-                padding: 1.6rem 2.4rem;
-                height: calc(100% - 200px);
-                max-height: 636px;
-            }
-        }
+    &--is-notifications-container-panel-open {
```

Comment:
as I commented above, can we change this accordingly?

Reply:
sure